docs/DeveloperGuide.md

```diff
@@ -269,64 +269,161 @@ _{Explain here how the data archiving feature will be implemented}_
-| `* * *`  | user                                       | delete a person              | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name        | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name         | locate a person easily                                                 |
+| Priority | As a …​  | I want to …​                              | So that I can…​                                                   |
+|---------|----------|-------------------------------------------|-------------------------------------------------------------------|
+| `* * *` | TA       | add a class                               | -                                                                 |
+| `* * *` | TA       | remove a class                            | -                                                                 |
+| `* * *` | TA       | view a class                              | view all students of a specific class                             |
+| `* * *` | TA       | add a student                             | -                                                                 |
+| `* * *` | TA       | remove a student                          | -                                                                 |
+| `* * *` | TA       | save the data automatically               | ensure that my data is not lost when I forget to save             |
+| `* *`   | TA       | update a class                            | keep the class information up to date                             |
+| `* *`   | TA       | update a student                          | keep the student record up to date                                |
+| `* *`   | TA       | take attendance in a lesson               | spend more time teaching                                          |
+| `* *`   | TA       | monitor a lesson of a class               | keep track of administrative information for each lesson          |
+| `* *`   | TA       | find a specific student by using a filter | quickly update their record such as attendance when they are late |
+| `* *`   | new user | see the app populated with sample data    | easily see what the app looks like when it is in use              |
 
 *{More to be added}*
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `EduTrack` and the **Actor** is the `user`, unless specified otherwise)
+
+**Use case: Add a class**
```

Comment:
May need to add System and actor as well I think
### This is a short snippet of what I pick out from the textbook

System: Online Banking System (OBS)
Use case: UC23 - Transfer Money
Actor: User
MSS: